tests/services/inventory/test_data/additional_cai_iam_policies.dump

```diff
@@ -2,8 +2,10 @@
 # enabled. These resources are merged into mock_cai_iam_policies.dump by the
 # update_cai_dumps.py script. Any lines that start with a '#' are ignored.
 #
-# Bigquery Dataset
+# Bigquery Dataset with project number
 {"name":"//bigquery.googleapis.com/projects/1042/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset","iam_policy":{"bindings":[{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project2"]},{"role":"roles/bigquery.dataOwner","members":["projectOwner:project2","user:user@forseti.testing"]},{"role":"roles/bigquery.dataViewer","members":["allAuthenticatedUsers","domain:forseti.testing","projectViewer:project2"]}]}}
+# Bigquery Dataset with project id
+{"name":"//bigquery.googleapis.com/projects/bigquerytestdefault/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset","iam_policy":{"bindings":[{"role":"roles/bigquery.dataEditor","members":["group:test-group@forseti.testing","projectEditor:project2"]},{"role":"roles/bigquery.dataOwner","members":["projectOwner:project2","user:user@forseti.testing"]},{"role":"roles/bigquery.dataViewer","members":["allAuthenticatedUsers","domain:forseti.testing","projectViewer:project2"]}]}}
```

Comment:
bigquerytestdefault needs to be project2 to match the other data that is used by the crawler.
bq_test_ds should be a different name, like bq_test_ds2

Reply:
Updated to use project3 instead, we are safe to assume only one style is being used at one time, right? (e.g. either project id or project number) If that's the case then updating it to project 2 will only return 1 result unless we do an append for both resource name format.